File: PYTHON/calculator.py
y=0
l=[int(input("")),input(""),int(input(""))]
#while(y<3):
    #l[y]=input("")
    #print(' ',end="")
    #y=y+1
no=float(input("enter equation : "))
op=input('')
no2=float(input(""))
x=0
while(x<1):
    if(op=='+'):
        no=no+no2
        print(no)
        op=input('')
        no2=float(input(""))
    elif(op=='-'):
        no=no-no2
        print(no)
        op=input('')
        no2=float(input(""))
    elif(op=='*'):
        no=no*no2
        print(no)
        op=input('')
        no2=float(input(""))
    elif(op=='/'):
        no=no/no2
        print(no)
        op=input('')
        no2=float(input(""))
    else:
        print("wrong input")
        x=1

#print(no,op,no2)

File: c programming/calculator.cpp
//calulator program on 2 no.s :-

#include <stdio.h>

int main()

{
	int n1,n2;
    char sign;
	
	printf("enter equation : ");
	scanf("%d%c%d",&n1,&sign,&n2);
	
	switch(sign)
	{
		case'+':printf("%d",n1+n2);break;
		case'-':printf("%d",n1-n2);break;
		case'*':printf("%d",n1*n2);break;
		case'/':printf("%d",n1/n2);break;
		
	}
	
	return 0;
}
